client/client.go

```diff
@@ -1065,7 +1065,7 @@ func (c *Client) updateNodeFromDriver(name string, fingerprint, health *structs.
 
 				// Only emit an event if the health status has changed, not if we are
 				// simply updating a node on startup
-				if health.Healthy != oldVal.Healthy && oldVal.HealthDescription != "" {
+				if health.Healthy != oldVal.Healthy && health.HealthDescription != "" {
```

Comment:
Is the comment still accurate? I'm not sure how this part:

> not if we are simply updating a node on startup

maps to the new check:

> `health.HealthDescription != ""`

Reply:
The comment is still accurate- basically, if the node is initializing, we don't want to log a node event. After initialization and the health status has changed, we do. I can clarify that comment though, thanks for pointing that out!